content/dev-tools/rpc-tool.page.tsx

```diff
@@ -0,0 +1,552 @@
+                        </Link>
+                      </li>
+                      <li>
+                        <Link
+                          id="explorerlink"
+                          to={`https://livenet.xrpl.org/${inputType}/${inputText}`}
+                          target="_blank"
+                        >
+                          {translate("Explorer")}
+                        </Link>
+                      </li>
+                    </ul>
+                  )}
+                  {txResponse && (
+                    <div className="group group-tx">
+                      <h3>
+                        <a href="tx.html">tx</a>
+                      </h3>
+                      <ul className="tools nav nav-pills">
+                        <li>
+                          <a className="tx-expand">{translate("expand all")}</a>
+                        </li>
+                        <li>
+                          <a className="tx-collapse">
+                            {translate("collapse all")}
+                          </a>
+                        </li>
+                      </ul>
+                      <ReactJson
+                        src={txResponse}
+                        name={false}
+                        theme="monokai"
+                        collapsed={1}
+                        collapseStringsAfterLength={100}
+                        displayDataTypes={false}
+                        displayObjectSize={false}
+                        enableClipboard={false}
+                        indentWidth={24}
+                        style={{ overflow: 'scroll' }}
+                      />
+                      <ul className="tools nav nav-pills">
+                        <li>
+                          <a className="tx-expand">{translate("expand all")}</a>
+                        </li>
+                        <li>
+                          <a className="tx-collapse">
+                            {translate("collapse all")}
+                          </a>
+                        </li>
+                      </ul>
+                    </div>
+                  )}
+                  <div className="group group-account">
+                    {accountInfoResponse && (
+                      <>
+                        <h3>
+                          <a href="account_info.html">account_info</a>
+                        </h3>
+                        <ReactJson
+                          src={accountInfoResponse}
+                          name={false}
+                          theme="monokai"
+                          collapsed={1}
+                          collapseStringsAfterLength={100}
+                          displayDataTypes={false}
+                          displayObjectSize={false}
+                          enableClipboard={false}
+                          indentWidth={24}
+                          style={{ overflow: 'scroll' }}
+                        />
+                      </>
```

Comment:
These ReactJson components are very complex, and seemingly mostly the same between different cases - would it make sense to componentize these sections with a parameter or two to do the small config changes?